realm/realm-library/src/main/java/io/realm/internal/PendingRow.java

```diff
@@ -58,12 +58,12 @@ public void onChange(PendingRow pendingRow) {
                         Row row = returnCheckedRow ? CheckedRow.getFromRow(uncheckedRow) : uncheckedRow;
```

Comment:
we should update the comment a few lines above.

`// If no rows returned by the query, just wait for the query updates until it returns a valid row.`